cob_people_detection/ros/launch/online_face_recognizer.launch

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+
+<launch>
+ <arg name="camera_namespace" default=""/>
+ <arg name="in_topic_name" value="/face_detector/face_positions"/>
+ <arg name="recognition_method" />
+
+  <!-- learn and recognizes faces online and publish their positions -->
+  <node name="online_face_recognizer" pkg="cob_people_detection" type="online_face_recognition_node.py" output="screen">
+    <rosparam command="load" file="$(find cob_people_detection)/ros/launch/online_face_recognizer_params.yaml"/>
+    <remap from="~face_detector/face_positions" to="$(arg camera_namespace)$(arg in_topic_name)"/>
```

Comment:
It seems like your pull request does not affect the camera_namespace argument. Just remove it or make a systematic proposal in all files. But for that start a new pull request.